.github/workflows/npmpublish.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           node-version: 12
           registry-url: https://npm.pkg.github.com/
           scope: '@leviy'
+      - run: npm version ${{github.event.release.name}}
```

Comment:
A disadvantage I can imagine of this change is that the version in the package.json of a checkout of this repository will still have the old version in it. Whilst the version on pkg.github.com has a different version as that one is bumped by this command.

A quick fix for this that I can think of is to specify version `0.0` in the package.json so that it's clear that it is incorrect.

Reply:
Correct. I also thought about the "`0.0`" solution. Maybe I'm going to change, now you mentioned it.

Reply:
Done, let's see if this will work 😄